backend/tests/test_background_worker_build.py

```diff
@@ -695,13 +695,13 @@ class _SideEffect:
         def __call__(self):
             self.counter += 1
             if self.counter == 1:
-                return (1, "err stdout", "err stderr")
-            return (0, "", "")
+                return (1, b"err stdout", "err stderr")
+            return (0, b"", "")
```

Comment:
How does stdout come out as a binary output?

Reply:
That was a leftover from before I used `run_expensive`, thanks for catching that